before-script.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
This file should be executable so that building doesn't generate changes.

Reply:
Ah. I was thinking the same, but I actually don't know how to do that? Do I need to change permissions to 777? Else, can you do that on your Mac, and push changes into this PR? If yes, that will be really great :-)